gtsam/slam/RegularImplicitSchurFactor.h

```diff
@@ -163,9 +171,12 @@ class RegularImplicitSchurFactor: public GaussianFactor {
         // (1 x 1) = (1 x 3) * (3 * 3) * (3 x 1)
         dj(k) -= FtE.row(k) * PointCovariance_ * FtE.row(k).transpose();
       }
-      d.insert(j, dj);
+
+      auto result = d.emplace(j, dj);
+      if(!result.second) {
+        result.first->second += dj;
```

Comment:
Nice! Is this the only other factor left? I thought there were some other variants of this factor around.

Reply:
Yes, because if not implemented it will bug out at compile time.